aspnetcore/signalr/redis-backplane.md

```diff
@@ -212,9 +212,14 @@ services.AddSignalR()
 
 :::moniker-end
 
-## Redis Clustering
+## Redis Cluster
+
+[Redis Cluster](https://redis.io/topics/cluster-spec) utilizes multiple simultaneously active Redis servers. When Redis Cluster is used as the backplane for SignalR, all messages are delivered to all of the nodes of the cluster without code modifications to the app. However, there's a tradeoff in that increasing the number of nodes in the cluster decreases the message throughput of the backplane. This is because each message must be sent to every active node in the cluster.
```

Comment:
Tradeoffs usually have two sides, this explains the downside, but not the upside.

Reply:
Yes, it's the opposite (decrease nodes = increase throughput) ... and the paragraph is calling out what happens when they're added. However, I think I can reword this to address it. *Stand-by .......*

Reply:
No, I think the tradeoff is that you get higher availability

Reply:
Oh ... I misunderstood what you meant. I'll make another patch to address it as "higher availability."

Reply:
I'm not sure if I'm making this *better* or ***worse***. You might want to suggest the exact text that you'd like.

I think I detect that a separate paragraph makes sense there tho, so split it out on the last commit to see how it composes.

Reply:
I added "achieve high availability" back in the first sentence so that the upside is clear from the start. It got dropped off in the iterations.